raiden-dapp/src/components/account/backup-state/UploadStateDialog.vue

```diff
@@ -136,14 +137,24 @@ export default class UploadStateDialog extends Vue {
     }, 2000);
   }
 
-  uploadStateAndConnect(uploadedFile: FileList) {
+  async connectAndRedirect(retrievedState: string) {
```

Comment:
Nitpick: 
Could you eventually move this function down? This will only be called by the `uploadState()` function which gets defined after, if I'm not wrong. At least I prefer to have the functions in a module a bit more sorted in the order they are used (some linter even force that).

Reply:
Cool, yes I will do that.